.github/workflows/generate-user-stories.yml

```diff
@@ -1,8 +1,6 @@
 name: publish 'in productie' en 'in ontwikkeling' user stories
```

Comment:
Ik denk dat het verstandige is om de versie van dit bestand in de master te handhaven. Die in de develop is volgens mij een oudere versie.

Reply:
klopt. Dat is nu gefixt met een merge vanuit main